src/SelfService/Infrastructure/Api/Capabilities/CapabilityController.cs

```diff
@@ -775,7 +775,7 @@ public async Task<IActionResult> GetCapabilityMetadata(string id)
         return Ok(metadata);
     }
 
-    [HttpPost("{id}/metadata")]
+    [HttpPut("{id}/metadata")]
```

Comment:
Why the `Put`?
Are we not using `Post` for most other total updates.
Or are we doing a partial update here?